cvat/apps/auto_annotation/model_manager.py

```diff
@@ -29,7 +29,7 @@ def _remove_old_file(model_file_field):
         os.remove(model_file_field.name)
 
 def _update_dl_model_thread(dl_model_id, name, is_shared, model_file, weights_file, labelmap_file,
-        interpretation_file, run_tests, is_local_storage, delete_if_test_fails):
+        interpretation_file, run_tests, is_local_storage, delete_if_test_fails, owner_id, run_as_admin=False):
```

Comment:
Why do we need owner_id here?

Reply:
In order to update the new owner of the model. If an admin updates an existing model, this is the only way to update the existing model with new privileges.